src/main/scala/Dao/DAO.scala

```diff
@@ -1,21 +1,21 @@
 package Dao
 
-import java.util.UUID
-import scala.collection.mutable.ListBuffer
 import models.Users
+
+import java.sql.ResultSet
 import scala.concurrent.Future
 
 trait DAO {
 
   def addUser(user: Users): Future[String]
 
-  def getById(userId: UUID): Future[Option[Users]]
+  def getById(userId: Int): Future[ResultSet]
```

Comment:
Why are you using Int type here? Why not UUID?

Reply:
I have used Int because DB is returning Integer Value.